native/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -0,0 +1,147 @@
+ */
+
+@CommandLine.Command(
+        name = "init",
+        description = "generate database configurations.")
+
+public class Init implements BLauncherCmd {
+    private final PrintStream outStream = System.out;
+    private final PrintStream errStream = System.err;
+    private String sourcePath = "";
+    private ProjectEnvironmentBuilder projectEnvironmentBuilder;
+    private String configPath = PersistToolsConstants.CONFIG_PATH;
+
+    private CommandLine parentCmdParser;
+    private String name = "";
+
+    Project balProject;
+
+    @CommandLine.Option(names = {"-h", "--help"}, hidden = true)
+    private boolean helpFlag;
+
+    @Override
+    public void execute() {
+        try  {
+            if (projectEnvironmentBuilder == null) {
+                balProject = ProjectLoader.loadProject(Paths.get(""));
+
+            } else {
+                balProject = ProjectLoader.loadProject(Paths.get(this.sourcePath), projectEnvironmentBuilder);
+            }
+            String orgName = balProject.currentPackage().descriptor().org().value();
+            String projectName = balProject.currentPackage().descriptor().name().value();
+            name = orgName + "." + projectName;
+        } catch (ProjectException e) {
+            errStream.println("The current directory is not a Ballerina project!");
+            return;
+        }
+        if (!Files.exists(Paths.get(sourcePath, configPath))) {
+            try {
+                createConfigToml();
+            } catch (Exception e) {
+                errStream.println("Failure when creating the Config.toml file: " + e.getMessage());
+            }
+        } else {
+            try {
+                updateConfigToml();
+            } catch (Exception e) {
+                errStream.println("Failure when updating the Config.toml file: " + e.getMessage());
+            }
+        }
+    }
+    private void createConfigToml() throws Exception {
+        SyntaxTree syntaxTree = CreateSyntaxTree.createToml();
+        writeOutputFile(syntaxTree, Paths.get(this.sourcePath, "Config.toml").toAbsolutePath().toString());
+    }
+
+    private void updateConfigToml() throws Exception {
+        SyntaxTree syntaxTree = CreateSyntaxTree.updateToml(Paths.get(this.sourcePath, this.configPath), this.name);
+        writeOutputFile(syntaxTree, Paths.get(this.sourcePath, "Config.toml").toAbsolutePath().toString());
+    }
+
+    private void writeOutputFile(SyntaxTree syntaxTree, String outPath) throws Exception {
+        String content = "";
+        Path pathToFile = Paths.get(outPath);
+        Files.createDirectories(pathToFile.getParent());
+        try {
+            content = Formatter.format(syntaxTree.toSourceCode());
+        } catch (FormatterException e) {
+            throw e;
+        }
+        try (PrintWriter writer = new PrintWriter(outPath, StandardCharsets.UTF_8.name())) {
+            writer.println(content);
+        } catch (IOException e) {
+            throw e;
+        }
```

Comment:
```suggestion
        content = Formatter.format(syntaxTree.toSourceCode());
        try (PrintWriter writer = new PrintWriter(outPath, StandardCharsets.UTF_8.name())) {
            writer.println(content);
        } catch (IOException e) {
            throw e;
        }
```

Reply:
No need to catch since the exception is immediately re-thrown.

Reply:
Updated in [e4c809d](https://github.com/ballerina-platform/persist-tools/pull/9/commits/e4c809da564408437e379bbe491d495ca2cbd2e8)